contracts/feeds/customfeeds/PriceFeedyVaultDAI.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.0;
+
+import "../../../interfaces/IPriceFeeds.sol";
+import "../../interfaces/IyVault.sol";
+
+contract PriceFeedyVaultDAI {
```

Comment:
since number of our pricefeeds growing. I would love to see standartization if pricefeed naming.

Reply:
The names relate to the asset being added. Some of the names are incredibly long so are truncated to convey what the asset is without being incredibly long on the contract name